pkg/ccl/changefeedccl/avro.go

```diff
@@ -623,6 +623,16 @@ func typeToAvroSchema(typ *types.T) (*avroSchemaField, error) {
 				return tree.ParseDTSVector(x.(string))
 			},
 		)
+	case types.PGVectorFamily:
+		setNullable(
+			avroSchemaString,
```

Comment:
I deliberately removed this code, though I probably should have commented as such, on advice from @asg0451. I assume it'll be more annoying to migrate to a new Avro format later, and stringifying the vectors is probably too inefficient. I think it would be better to leave this unimplemented, or alternatively implement it as a nicely packed binary number array.

Reply:
just curious, what will happen if a user creates a changeed on a table with a vector? will it fail to create? fail to run? silently ignore that column?

Reply:
I'm assuming that it'll be possible to create the changefeed, but then it'll fail with a terminal error when trying to encode the event into the appropriate format.

Reply:
can we test this assumption real quick? then we can ask @kathancox to document this known limitation.

Reply:
It's probably going to be easier for someone on CDC to try then for me. I tried doing something like
```sql
CREATE TABLE t (v VECTOR);
INSERT INTO t VALUES('[1]'), ('[2,3]');
CREATE CHANGEFEED FOR t INTO 'null://sink' WITH format='avro';
```
but I'm hitting some errors. I'll leave it up to you @asg0451 to try it and file an issue if necessary.